### PR TITLE
Update disaster_recovery.rst

### PR DESCRIPTION
**Pull request by nrauso**
Added mandatory step for disaster recovery from Cockpit.

### CODE REVIEW

**Review by gsanchietti — CHANGES_REQUESTED** [review]
The `Reinstall packages` check is not mandatory.
Let's say you want to restore a firewall, but without web proxy, you can just uncheck the option and reinstall required packages at the end of the restore.

File: administrator-manual/en/disaster_recovery.rst [hunk 1]
```diff
@@ -59,15 +59,18 @@ Please, follow below steps:
 5. Map network interface names from the backup to the running system.
    This step is required only if :guilabel:`Restore network configuration` option is enabled.
 
-6. Click the :guilabel:`Restore` to start the restore process.
+6. It's **mandatory** to keep the option :guilabel:`Reinstall packages` **enabled**!
+   Its deactivation could introduce unexpected behavior on restored system!
```
Comment (gsanchietti) [inline administrator-manual/en/disaster_recovery.rst:63]:
```suggestion
6. During disaster recovery, to obtain a machine identical to original one, it's **mandatory** to keep the option :guilabel:`Reinstall packages` **enabled**!
   Its deactivation could introduce unexpected behavior on restored system!
```

Comment (nrauso) [reply administrator-manual/en/disaster_recovery.rst:63]: Ok for me

Comment (nrauso) [reply administrator-manual/en/disaster_recovery.rst]: We have to consider that disaster recovery is a "special" version of restore, both for config and data: you want to replicate of the scenario you had before.
If you want a partial restore it is not a complete disaster recovery.

**Pull request merged**